### PR TITLE
DEV-46 Review registration and renewal emails to ensure alignment wit…

### PR DESCRIPTION
**Pull request by moseshll**
…h service description

- Update registration e-mail based on input from HathiTrust staff.

### CODE REVIEW

File: app/views/shared/_registration_link.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<a href="<%= finalize_url(@registration.token, host: @base_url) %>"
+<a href="<%= finalize_url(@registration.token, host: @base_url, locale: nil) %>"
    style="font-weight:bold; font-family:courier; font-size:120%; color:#d65d22">
```
Comment (mwarin) [inline app/views/shared/_registration_link.html.erb:2]: I'd pick `class="..."`  over `style="..."` .

**Review by mwarin — APPROVED** [review]
Looks OK.

File: app/views/layouts/mailer.html.erb [hunk 2]
```diff
@@ -1,12 +1,17 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (mwarin) [inline app/views/layouts/mailer.html.erb:2]: Should be `<%= I18n.locale %>` maybe?

Comment (moseshll) [reply app/views/layouts/mailer.html.erb:2]: It's a W3C recommendation to have the tag, so it shuts up the pedantic warnings :-)
But actually, no we don't want our emails to (necessarily) be in the same language as the UI language, they're orthogonal. It's a can I/we deliberately kicked down the road when I did the wacko I18n branch. One that will prolly be painful when we revisit it.

Comment (moseshll) [reply app/views/layouts/mailer.html.erb:2]: Actually it won't be that painful. When we have multiple languages for our outgoing e-mails we will have a "flag" menu in the preview page. How we're to know which language to send is anyone's guess, however.

Comment (mwarin) [reply app/views/layouts/mailer.html.erb:2]: `email_lang = I18n.locales.sample`

**Pull request merged**